apps/alpha-test-app/src/components/molecules/LayoutDemoOptionsMolecule.vue

```diff
@@ -0,0 +1,59 @@
+<script lang="ts" setup>
+import { computed } from "vue";
+
+type Options = Record<string, boolean | string>;
+
+const props = defineProps<{
+  modelValue: Options;
+  highlightLabel?: string;
+}>();
+
+const emit = defineEmits<{
+  "update:modelValue": [value: Options];
+}>();
+
+const options = computed({
+  get: () => props.modelValue,
+  set: (value) => emit("update:modelValue", value),
+});
+</script>
+
+<template>
+  <div class="options">
+    Demo options
+    <template v-for="(value, label) of options" :key="label">
+      <label v-if="typeof value === 'boolean'">
```

Comment:
We can use our OnyxCheckbox :)

Reply:
is it ok if I do it in the[ follow-up ticket](https://github.com/SchwarzIT/onyx/issues/374)? I wanted to make a very basic initial draft.
I'll add it to the AC